web-ui/src/main/resources/catalog/views/default/templates/recordView.html

```diff
@@ -317,9 +317,50 @@ <h2 data-translate="">aboutThisResource</h2>
 
           <h2 data-translate="">technicalInformation</h2>
 
+          <div data-ng-if="mdView.current.record.featureTypes"
+            data-ng-repeat="featureType in mdView.current.record.featureTypes">
+
+            <h3><span data-translate="">featureType</span>&nbsp;'{{featureType.typeName}}'</h3>
+
+            <table class="table table-striped" >
+              <tbody>
+              <tr data-ng-if="featureType.typeName">
+                <th data-translate="">featureTypeName</th>
+                <td>{{featureType.typeName}}</td>
+              </tr>
+              <tr data-ng-if="featureType.definition">
+                <th data-translate="">featureDefinition</th>
+                <td>{{featureType.definition}}</td>
+              </tr>
+              <tr data-ng-if="featureType.code">
+                <th data-translate="">featureCode</th>
+                <td>{{featureType.code}}</td>
+              </tr>
+              <tr data-ng-if="featureType.isAbstract">
+                <th data-translate="">featureIsAbstract</th>
+                <td>{{featureType.isAbstract}}</td>
+              </tr>
+              <tr data-ng-if="featureType.aliases">
+                <th data-translate="">featureAliases</th>
+                <td>{{featureType.aliases}}</td>
+              </tr>
+              <tr data-ng-if="featureType.attributeTable">
+                <th data-translate="">featureAttributeTable</th>
+                <td></td>
+              </tr>
+              <tr data-ng-if="featureType.attributeTable">
+                <td colspan="2" class="gn-noborder-top gn-nopadding-top gn-nopadding-bottom"/>
+              </tr>
+              <tr data-ng-if="featureType.attributeTable">
+                <td colspan="2" class="gn-noborder-top">
+                  <div data-ng-if="featureType.attributeTable"
```

Comment:
Making a table in a table make the layout a bit hard to read no ? @MichelGabriel do you have ideas ?
I would make one table for feature type characteristics and another for its attribute maybe ?

Reply:
I guess a table in a table is ok when done as shown in the attached image/screenshot. The table in a table has another style (bordered). I can make it look more like a list than a table.

The thing that strikes me the most is that there is a lot of padding/margin around the table in table. That shouldn't be there.

Reply:
Indeed, on a first sight, those margins and padding are striking. They come here from bootstrap, I did not want to add too much custom styling.

Also, I find them useful to structure the display. Some metadata have a lot of features, some with a lot of attributes, and some of those attributes with long value list. Removing the margins and padding in those cases leads to a page where IMO it is not so easy to figure out which information belongs to what (feature or attribute).

So I think also those margins&padding are not ideal. At the same time, if they are to be removed, I would suggest to find another way to structure the information, the goal being to easily distinguish what informs about the attributes and what informs about the features.